dev-python/email-validator/email-validator-1.1.0.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+DISTUTILS_USE_SETUPTOOLS=rdepend
+
+inherit distutils-r1
+
+MYPN="python-${PN}"
```

Comment:
Why are you renaming it in the first place? This is just confusing to everyone, and will become trouble once someone independently creates `email-validator` package (the name is free, isn't it?).